src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -251,11 +251,10 @@ private int countEntitiesInDatastore(String lectureKeyId) {
     return datastore.prepare(filteredQuery(lectureKeyId)).countEntities(withLimit(100));
   }
 
-  private Query filteredQuery(String lectureKeyId) {
-    Key lectureKeyKey =
-        KeyFactory.createKey(TranscriptServlet.PARAM_LECTURE, Long.parseLong(lectureKeyId));
-    Filter lectureKeyFilter =
-        new FilterPredicate(TranscriptLine.PROP_LECTURE, FilterOperator.EQUAL, lectureKeyKey);
-    return new Query(TranscriptLine.ENTITY_KIND).setFilter(lectureKeyFilter);
+  private Query filteredQuery(String lectureId) {
```

Comment:
I notice you're passing the ID around as a string and calling parseLong within each method.  It might be better to parse the ID once and pass the long (or resulting Key) around instead.  This applies outside of tests too.